graphql/http_handler.go

```diff
@@ -57,12 +57,14 @@ func Handler(apiURL string) func(w http.ResponseWriter, r *http.Request) {
 			queryPath = fieldCtx.Path().String()
 			args = fieldCtx.Args
 		}
-		grip.Error(message.WrapError(err, message.Fields{
-			"path":    "/graphql/query",
-			"query":   queryPath,
-			"args":    args,
-			"request": gimlet.GetRequestID(ctx),
-		}))
+		if !errors.Is(err, context.Canceled) {
+			grip.Error(message.WrapError(err, message.Fields{
```

Comment:
How do you feel about keeping these around and just logging at a lower level if they exist? Just in case a context cancelled error ever appears we could have some logs to use for an investigation instead of throwing away everything.

Reply:
I commented this above:

> I thought we'd originally discussed not logging this, since it does still log 1000s of logs we don't intend to look at. Do we have actual planned follow up work to do something with these logs, with a ticket we can include in the message? Otherwise it doesn't make sense to log these until we do.
> 
> We could also possibly do something like
> 
> https://github.com/evergreen-ci/evergreen/blob/91095164b387b5216b26082e0b830cdb38fc2b16/rest/data/repotracker.go#L45
> 
> to avoid logging in a noisy way.

I'm pretty against logging something that logs this often without explicit planned follow up work.

Reply:
*explicit planned follow up OR a specific alert and guidance on how to approach it